script.js

```diff
@@ -0,0 +1,200 @@
+            "summaryTwo": "Revenue had positive growth each quarter, our education and events had a higher number of attendees, and an increasing number of spas and salons were taking on our brands."
+        },
+    },
+    {
+        "company": "Continental Cosmetics Ltd",
+        "position": "Customer Support Expert",
+        "startDate": "May 2018",
+        "endDate": "December 2018",
+        "summary": {
+            "summaryOne": "Built relationships with customers via phone, email, and in person to ensure spas and salons had everything they needed to run a successful business.",
+            "summaryTwo": "Through building close relationships with customers, I was able to find what brands, products and education were a good fit for each individual business."
+        },
+    },
+{
+    "company": "LifeSpa - LifeTime Fitness",
+        "position": "Guest Service Manager",
+        "startDate": "May 2017",
+        "endDate": "May 2018",
+        "summary": {
+            "summaryOne": "Managed a team of Spa Coordinators whose main goal was to guide guests through their entire spa visit, from beginning to end.",
+            "summaryTwo": "Utilizing internal software to manage guest book, staff scheduling, POS sales, inventory and product ordering allowed us to control revenue stream."
+        },
+}],
+    "education": [{
+        "institution": "Juno College of Technology",
+        "area": "Web Development",
+        "studyType": "Certificate",
+        "date": "Winter 2020",
+        "courses": [
+            "Web Development Fundamentals",
+            "JavaScript Fundamentals",
+            "Web Development Immersive"
+        ]
+    },
+{
+    "institution": "Seneca College",
+        "area": "Esthetics and Spa Management",
+        "studyType": "Diploma",
+        "date": "Fall 2014 - Spring 2017",
+        "courses": [
+            "Customer Service",
+            "Business ManagementT"
+        ]
+}],
+    "skills": [{
+        "name": "Web Development",
+        "level": "Master",
+        "keywords": [
+        "HTML",
+        "CSS",
+        "Javascript / jQuery",
+        "React",
+        "Working with APIs",
+        "Firebase",
+        "Web Accessibility",
+        "Git / GitHub",
+        "Email Development",
+        "Wireframing",
+        "PSD Conversion",
+        "Responsive Design",
+        "Sketch / Figma",
+        "Adobe Photoshop",
+        "Adobe Illustrator"
+        ]
+    }],
+    "languages": [{
+        "language": "English",
+        "fluency": "Native speaker"
+    }]
+}
+
+let name = resume.basics.name
+let profession = resume.basics.label
+let website = resume.basics.website
+let phone = resume.basics.phone
+let email = resume.basics.email
+let summary = resume.basics.summary
+let resumeSkills = resume.skills[0].keywords
+let skillsToPrint = ""
+let resumeEducation = resume.education
+let educationToPrint = ""
+let resumeExperience = resume.work
+let experienceToPrint = ""
```

Comment:
I think if we are not planning on modifying some of these values, perhaps we can use 'const' instead of 'let'.